lib/cmd/common/preview.js

```diff
@@ -49,10 +51,10 @@ function doPreview(appId, cb) {
   exec(fhc.config.get("browser"), [url], function (err) {
     if (err) {
       log.er(cb,
-        "Failed to open " + url + " in a browser.  It could be that the\n" +
-        "'browser' config is not set.  Try doing the following:\n" +
-        "    fhc set browser google-chrome\n or:\n" +
-        "    fhc set browser lynx\n");
+             util.format(i18n._("Failed to open %s in a browser.  It could be that the\n" +
+				"'browser' config is not set.  Try doing the following:\n" +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace all tab characters in this file by sequences of white-spaces. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATabCharacter)